config/confighttp/confighttp.go

```diff
@@ -203,6 +203,9 @@ type HTTPServerSettings struct {
 	// Endpoint configures the listening address for the server.
 	Endpoint string `mapstructure:"endpoint"`
 
+	// Server Transport config. For transport only "tcp" / "tcp4" / "tcp6" are valid options.
+	Transport string `mapstructure:"transport"`
```

Comment:
We need unit tests for this.

Reply:
Is this comment correctly place in this thread? I don't feel that a unittest will break anything.

Reply:
This is not a unit test O_O

Reply:
Sorry, just realized I hijacked your comment. My comments are a different concern than yours.